airbyte-server/src/main/java/io/airbyte/server/handlers/WebBackendConnectionsHandler.java

```diff
@@ -368,12 +367,14 @@ public WebBackendConnectionRead webBackendGetConnection(final WebBackendConnecti
     return buildWebBackendConnectionRead(connection, currentSourceCatalogId).catalogDiff(diff);
   }
 
-  private Optional<SourceDiscoverSchemaRead> getRefreshedSchema(final UUID sourceId)
+  private Optional<SourceDiscoverSchemaRead> getRefreshedSchema(final UUID sourceId, final UUID connectionId)
```

Comment:
nit: since this function is actually going to fetch the schema everytime, feels like `refreshSchema` is more explicit. A bit out-of-scope, I'll probably rename this at some point.